include/rmm/mr/device/tracking_resource_adaptor.hpp

```diff
@@ -65,6 +65,47 @@ class tracking_resource_adaptor final : public device_memory_resource {
         allocation_size{size} {};
   };
 
+  /**
+   * @brief Stores the current, peak, and total number of allocation and
+   * allocated bytes
+   */
+  struct allocation_counts {
+    ssize_t current_bytes{0};    // Current outstanding bytes
```

Comment:
Hmmm, this  is the first I've seen `ssize_t`. Seems it's a C type, not C++, so let's not use it. Use int64_t or  `std::ptrdiff_t`.

Reply:
Fixed.